### PR TITLE
Multiple small bug fixes for SPLAT-626

### PR DESCRIPTION
**Pull request by bostrt**
Addressing a few fixes in this PR that are all related to logging.

- [x] SPLAT-624
- [x] SPLAT-628
- [x] SPLAT-630
- [x] SPLAT-638

### CODE REVIEW

File: pkg/run/run.go [hunk 1]
```diff
@@ -75,31 +76,39 @@ func NewCmdRun() *cobra.Command {
...
+				log.WithError(err).Fatal("error retrieving aggregator status")
 			}
 
-			// TODO Why's there a second StatusOptions instance?
-			st := status.NewStatusOptions(o.watch)
-			st.Update(sclient)
-			st.Print(cmd, sclient)
+			err = s.Update(sclient)
+			if err != nil {
+				log.Fatal(err)
```
Comment (rvanderp3) [inline pkg/run/run.go:101]: should a fatal error result in exiting this function?

File: pkg/run/run.go [hunk 2]
```diff
@@ -75,31 +76,39 @@ func NewCmdRun() *cobra.Command {
...
+			}
 
-			log.Info("Sonobuoy pods are ready!")
+			err = s.Print(cmd, sclient)
+			if err != nil {
+				log.Fatal(err)
```
Comment (rvanderp3) [inline pkg/run/run.go:106]: should a fatal error result in exiting this function?

File: pkg/run/run.go (hunk 1 above)
Comment (bostrt) [reply pkg/run/run.go:101]: Yeah, that's the intent where the `log.Fatal` were added in this PR

**Pull request merged**